src/__tests__/components/FilledTextInput.test.tsx

```diff
@@ -3,14 +3,14 @@ import * as React from 'react'
 import { createRenderer } from 'react-test-renderer/shallow'
 
 import { SearchIconAnimated } from '../../components/icons/ThemedIcons'
-import { FilledTextInput } from '../../components/themed/FilledTextInput'
+import { FilledTextInputRaw } from '../../components/themed/FilledTextInput'
```

Comment:
I'm not sure if there was team consensus around this, but I would invert this name change. FilledTextInput should be the "raw" version. It should be capable of being a library used outside of Edge.

`ModalFilledTextInput` would be the preferred name for a wrapper that adds padding specific to modals or whatever use Edge needs.

Reply:
My two considerations that prompted the change were:
1. Make it more apparent changes are needed to bring a parent up to spec within our codebase, to make sure it's painfully obvious it's not preferable for callers to make up their own special styling
2. That this layout isn't specific to modals but UI4 conventions in general of splitting 0.5rem everywhere

Mainly #2 drove this change but we can just duplicate the styling I suppose between modal and scene.